webapp/channels/src/components/user_settings/notifications/user_settings_notifications.tsx

```diff
@@ -147,9 +150,10 @@ function getDefaultStateFromProps(props: Props): State {
     let channelKey = false;
     let isCustomKeysWithNotificationInputChecked = false;
     const customKeysWithNotification: MultiInputValue[] = [];
+    const customKeysWithHighlight: MultiInputValue[] = [];
 
     if (props.user.notify_props) {
-        if (props.user.notify_props.mention_keys) {
+        if (props.user.notify_props.mention_keys.length > 0) {
```

Comment:
Should we add a null check here in case just in case? Before it was checking directly if it existed ? Also, why this change?

```suggestion
        if (props.user.notify_props.mention_keys?.length > 0) {
```

Reply:
changed